gst/droidcamsrc/gstdroidcamsrc.h

```diff
@@ -48,7 +48,7 @@ G_BEGIN_DECLS
 #define GST_IS_DROIDCAMSRC_CLASS(klass) \
   (G_TYPE_CHECK_CLASS_TYPE((klass), GST_TYPE_DROIDCAMSRC))
 
-#define MAX_CAMERAS 2
+#define MAX_CAMERAS 255
```

Comment:
Is this still needed? Would we ever not be able to support whatever android_media_camera_get_number_of_cameras declares?